src/pi-src/test_uid.cpp

```diff
@@ -0,0 +1,106 @@
+/******************************************************************************
+Test cases for UID decoding.
+
+Author:  Noah Strong
+Created: 2018-02-28
+******************************************************************************/
+
+#include "catch.hpp"
+
+#include "uid.h"
+#include "data_collection.h"
+
+float calc_ping_dur_us(int id){
```

Comment:
Can you add a comment explaining that "us" means microseconds? Or change "us" to "microsecs".

Reply:
Is there some central place I should put this? It's a fairly common suffix and I use it in a lot of places, so test code might not be the most intuitive place to look for a definition